docs/source/tutorial_material/tutorial.ipynb

```diff
@@ -119,12 +115,12 @@
     "\n",
```

Comment:
"Additionally, the attributes \n",
    "- `param_dim`\n",
    "- `data_dim`\n",
    "- `PARAM_LIMITS`\n",
    "- `CENTRAL_PARAM`\n",

See a few lines above. Maybe say that you need to pass central_param and param_limits to the constructor? PARAM_LIMITS and CENTRAL_PARAM are just for convenience in our example models. But that's just a small detail, which I noticed while looking over the tutorial.